scripts/apps/web-publisher/services/index.js

```diff
@@ -0,0 +1,2 @@
+export { PublisherFactory } from './publisher-factory';
```

Comment:
We generally use to name these files `PublisherFactory.js` and `PubAPIFactory.js` in the rest of the project. Your approach is not wrong, but this is mostly how `core` is built, and not `apps`. It would be nice if you could follow the same pattern.